web/404.html

```diff
@@ -1,9 +1,24 @@
 <!doctype html5>
 <html>
   <head>
-    <title>Haas Hall Academy — What are you doing?!</title>
+    <title>404 - Page Not Found</title>
   </head>
+  <style>
+    body {
+    background-color:rgb(1,81,154);
+    font-family:"Palatino Linotype", serif;
```

Comment:
The `Palatino Linotype` font isn't showing up for me? Serif is, though, but the font might have to be installed in the site if it doesn't show up for the user? ☹️